test/cli/simple-package-rbi-gen/test.out

```diff
@@ -1,6 +1,12 @@
 No errors! Great job.
 # typed: true
 
+module ::Project
+end
+
+module ::Project::Foo
+end
+
```

Comment:
I don't have a ton of context on the rbi gen stuf going on here, but this seems like effectively a noop. cc @aadi-stripe @jez

Reply:
Doesn't this just mean that because these symbols are not available at the top-level by default anymore (due to the re-namespacing), we end up generating the shims for them?

Reply:
seems fine to me, we can figure it out later if it's a problem

Reply:
Yep, that's my understanding.